source/includes/_api_v3.md

```diff
@@ -0,0 +1,42 @@
+# Cloudify REST API V3.0
+
+> `every request require user credentials and tenant`
+
+```python
+# include this code when using cloudify python client-
+from cloudify_rest_client import CloudifyClient
+client = CloudifyClient(
+        host='<manager-ip>',
+        username='<manager-username>',
+        password='<manager-password>',
+        tenant='<manager-tenant>')
+```
+
+```html
+obsolete
+```
+
+Cloudify 4.0.0 introduced REST API V3.0.
+
+The base URI for the v3.0 REST API is: `/api/v3.0`.
+
+Starting from Cloudify 4.0.0, all communication to the server requires:
+ * Authentication using user credentials.
+ * Tenant name, representing the scope of the request.
+
+Every Cloudify Manager have a default tenant, called `default_tenant`.
+The `default_tenant` tenant is being created during bootstrap.
+
+In addition to the `default_tenant`, every Cloudify Manager have a bootstrap Admin.
```

Comment:
In addition to the `default_tenant`, every Cloudify Manager ~~have~~ **includes** a bootstrap Admin.